audit/lambda/toFirehoseTester-app.ts

```diff
@@ -0,0 +1,30 @@
+import { config, Firehose } from "aws-sdk";
+
+export const handler = async (record: String): Promise<void> => {
+
+  config.update({ region: process.env.AWS_REGION });
+
+  const firehose = new Firehose({apiVersion: '2015-08-04'});
+
+  if (process.env.firehoseName){
+    var params: Firehose.PutRecordInput = {
+        DeliveryStreamName: process.env.firehoseName,
+        Record: { Data: record },
+    };
+  } else {
+    console.error("[ERROR] ENV VAR MISSING: \n missing firehose name enironment variable");
+    throw new Error("[ERROR] ENV VAR MISSING: \n missing firehose name enironment variable");
```

Comment:
thinking about it now, we may just be able to throw and not log? maybe validate this? Otherwise we can just log and 'return' here? Maybe double check best practice in regards to Lambda.

Just trying to avoid the double logging as you mentioned earlier.